CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - **[Breaking]** `ValueType` and `GetType` functionality is removed in favor of using
   `reflect.Kind`
+- Update documentation: cleanup markup, trim readme and doc.go in
```

Comment:
Probably not - the changelog should only contain things end users would want to know.